metadata_tools/common.py

```
@@ -0,0 +1,224 @@
+        if template_path:
+            self.template_path = FCPath(template_path)
+        self.volume_id = volume_id
+        self.level = level
+        self.qualifier = qualifier
+        self.use_global_template = use_global_template
+
+        if not output_dir:
+            return
+
+        if not suffix:
+            suffix = "_%s_%s.tab" % (self.qualifier, self.level)
```

Ruff (0.14.10)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)